vivo.owl

```diff
@@ -12467,6 +12469,23 @@ Was called IMPP in vCard.</rdfs:comment>
 
 
 
+    <!-- http://vivoweb.org/ontology/core#wikidataId -->
+
+    <owl:DatatypeProperty rdf:about="http://vivoweb.org/ontology/core#wikidataId">
+        <rdfs:label xml:lang="en">Wikidata ID</rdfs:label>
+        <obo:IAO_0000111 xml:lang="en">Wikidata ID</obo:IAO_0000111>
+        <obo:IAO_0000118 xml:lang="en">Wikidata identifier</obo:IAO_0000118>
+        <rdfs:comment xml:lang="en">A Wikidata ID is a unique identifier assigned by Wikidata to an entity. They are alphanumeric strings (e.g., Q28136655) that serve as the local identifiers for entities in the wikidata.org.</rdfs:comment>
+        <obo:IAO_0000112 rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Identifier being used by Wikidata to identify entities in Wikidata.org. Wikidata IDs for entitiies should follow the pattern http://www.wikidata.org/entity/ID where ID is an entity ID. Example: https://www.wikidata.org/wiki/Q28136655</obo:IAO_0000112>
+        <vitro:exampleAnnot rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Identifier being used by Wikidata to identify entities in Wikidata.org. Wikidata IDs for entitiies should follow the pattern http://www.wikidata.org/entity/ID where ID is an entity ID. Example: https://www.wikidata.org/wiki/Q28136655</vitro:exampleAnnot>
+        <obo:IAO_0000115 rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Each Wikidata entity is identified by an entity ID, which is a number prefixed by a letter. Entity IDs can also be used as globally unique URIs that follow the pattern https://www.wikidata.org/entity/ID where ID is an entity ID.</obo:IAO_0000115>
+        <obo:IAO_0000119 rdf:datatype="http://www.w3.org/2001/XMLSchema#string">https://www.wikidata.org/wiki/Wikidata:Identifiers</obo:IAO_0000115>
+        <rdfs:subPropertyOf rdf:resource="http://vivoweb.org/ontology/core#identifier"/>
+        <rdfs:isDefinedBy rdf:datatype="http://www.w3.org/2001/XMLSchema#anyURI">https://www.wikidata.org/wiki/Wikidata:Identifiers</rdfs:isDefinedBy>
```

Comment:
This should refer to the defining ontology, while 0119 refers to the intellectual source: ```<rdfs:isDefinedBy rdf:resource="http://vivoweb.org/ontology/core"/>```

Reply:
Thanks, fixed.